pkg/apis/kops/instancegroup.go

```diff
@@ -363,3 +365,9 @@ type LoadBalancer struct {
 	// TargetGroupARN to associate with this instance group (AWS ALB/NLB)
 	TargetGroupARN *string `json:"targetGroupARN,omitempty"`
 }
+
+// AcceleratorConfig defines an accelerator config
+type AcceleratorConfig struct {
+	AcceleratorCount int64  `json:"acceleratorCount,omitempty"`
+	AcceleratorType  string `json:"acceleratorType,omitempty"`
```

Comment:
Why not just `count` and `type`? Should be pretty obvious they refer to accelerators.
```suggestion
	Count int64  `json:"count,omitempty"`
	Type  string `json:"type,omitempty"`
```

Reply:
gcp fields of InstanceTemplate are named with the prefix `accelerator`. I think we need to keep uniformity.

Reply:
Not a big fan of this, but can live with it.